### ISSUE TITLE
Test failures due to version change

### ISSUE THREAD

**Issue by schlessera** [wp-cli/core-command#8]
The core command currently has several failures:

```
01. version	update_type	package_url
    4.4.10	minor	https://downloads.wordpress.org/release/wordpress-4.4.10-partial-0.zip
    4.7.5	major	https://downloads.wordpress.org/release/wordpress-4.7.5.zip
    In step `Then STDOUT should be a table containing rows:'. # vendor/wp-cli/wp-cli/features/steps/then.php:44
    From scenario `Check for update via Version Check API'.   # vendor/wp-cli/core-command/features/core-check-update.feature:3
    Of feature `Check for more recent versions'.              # vendor/wp-cli/core-command/features/core-check-update.feature
02. version	update_type	package_url
    4.0.18	minor	https://downloads.wordpress.org/release/wordpress-4.0.18-partial-0.zip
    In step `Then STDOUT should be a table containing rows:'.              # vendor/wp-cli/wp-cli/features/steps/then.php:44
    From scenario `No minor updates for an unlocalized WordPress release'. # vendor/wp-cli/core-command/features/core-check-update.feature:44
    Of feature `Check for more recent versions'.                           # vendor/wp-cli/core-command/features/core-check-update.feature
03. $ wp core update --minor
    Updating to version 3.7.21 (en_US)...
    Downloading update from https://downloads.wordpress.org/release/wordpress-3.7.21-partial-9.zip...
    Unpacking the update...
    Success: WordPress updated successfully.
    
    cwd: /tmp/wp-cli-test-run-591f3816366f34.23278218/
    exit status: 0
    In step `Then STDOUT should contain:'.                                 # vendor/wp-cli/wp-cli/features/steps/then.php:15
    From scenario `Update to the latest minor release'.                    # vendor/wp-cli/core-command/features/core-update.feature:34
    Of feature `Update WordPress core'.                                    # vendor/wp-cli/core-command/features/core-update.feature
04. $ wp core update --minor
    Updating to version 4.0.18 (en_US)...
    Descargando paquete de instalación desde https://downloads.wordpress.org/release/wordpress-4.0.18-partial-0.zip...
    Descomprimiendo actualización...
    <p>Algunas de tus traducciones necesitan actualizarse. Espera unos segundos más mientras las actualizamos también.</p>
    <div class="update-messages lp-show-latest"><h4>Actualizando traducciones de Twenty Fifteen (es_ES)&#8230;</h4><p>Descargando traducción desde <span class="code">https://downloads.wordpress.org/translation/theme/twentyfifteen/1.7/es_ES.zip</span>&#8230;</p>
    <p>Descomprimiendo actualización&#8230;</p>
    <p>Instalando última versión&#8230;</p>
    <p>Traducción actualizada con éxito.</p>
    </div><div class="update-messages lp-show-latest"><h4>Actualizando traducciones de Twenty Fourteen (es_ES)&#8230;</h4><p>Descargando traducción desde <span class="code">https://downloads.wordpress.org/translation/theme/twentyfourteen/1.2/es_ES.zip</span>&#8230;</p>
    <p>Descomprimiendo actualización&#8230;</p>
    <p>Instalando última versión&#8230;</p>
    <p>Traducción actualizada con éxito.</p>
    </div><div class="update-messages lp-show-latest"><h4>Actualizando traducciones de Twenty Seventeen (es_ES)&#8230;</h4><p>Descargando traducción desde <span class="code">https://downloads.wordpress.org/translation/theme/twentyseventeen/1.0/es_ES.zip</span>&#8230;</p>
    <p>Descomprimiendo actualización&#8230;</p>
    <p>Instalando última versión&#8230;</p>
    <p>Traducción actualizada con éxito.</p>
    </div><div class="update-messages lp-show-latest"><h4>Actualizando traducciones de Twenty Sixteen (es_ES)&#8230;</h4><p>Descargando traducción desde <span class="code">https://downloads.wordpress.org/translation/theme/twentysixteen/1.3/es_ES.zip</span>&#8230;</p>
    <p>Descomprimiendo actualización&#8230;</p>
    <p>Instalando última versión&#8230;</p>
    <p>Traducción actualizada con éxito.</p>
    </div><div class="update-messages lp-show-latest"><h4>Actualizando traducciones de Twenty Thirteen (es_ES)&#8230;</h4><p>Descargando traducción desde <span class="code">https://downloads.wordpress.org/translation/theme/twentythirteen/1.3/es_ES.zip</span>&#8230;</p>
    <p>Descomprimiendo actualización&#8230;</p>
    <p>Instalando última versión&#8230;</p>
    <p>Traducción actualizada con éxito.</p>
    </div><script type="text/javascript">
    					(function( wp ) {
    						if ( wp && wp.updates.decrementCount ) {
    							wp.updates.decrementCount( "translation" );
    						}
    					})( window.wp );
    				</script>Success: WordPress updated successfully.
    
    Warning: Failed to fetch checksums. Please cleanup files manually.
    cwd: /tmp/wp-cli-test-run-591f387cda1761.89821841/
    exit status: 0
    In step `Then STDOUT should contain:'.                                 # vendor/wp-cli/wp-cli/features/steps/then.php:15
    From scenario `Minor update on an unlocalized WordPress release'.      # vendor/wp-cli/core-command/features/core-update.feature:199
    Of feature `Update WordPress core'.                                    # vendor/wp-cli/core-command/features/core-update.feature
```

These seem to be due to a version change of WordPress core.

Related: https://github.com/wp-cli/core-command/issues/4

**Comment by gitlost** [comment]
The second one "core-update.feature:199" with the old "4.0.16" version got past travis due to a "require-wp-4.0" tag, which is never fulfilled in the current matrix.

The first one "core-update.feature:34" is weird - that output should match the expected "Updating to version 3.7.21", and I can't reproduce the failure either on travis or locally. Maybe some other exception took place?

**Comment by danielbachhuber** [comment]
> The core command currently has several failures:

Where? Builds are passing on master: https://travis-ci.org/wp-cli/core-command/builds/233912046

**Comment by gitlost** [comment]
The "core-update.feature:199" with the old "4.0.16" version needs to be fixed though, and updated to "4.0.18", and the "require-wp-4.0" tag removed. It's getting hit here as that tag isn't being set.

**Comment by websolutionvilla** [comment]
You can check all details about [WordPress latest version features ](https://websolutionvilla.com/whats-new-in-wordpress-4-8-version.html)
